### DISCUSSION TITLE
Your browser does not meet the security requirements for Kibana.

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#8150] [Unsupported Versions] [answered]
Does anyone else get the following warning in SO Version: 2.3.130? the browser is up to date too.

![image](https://user-images.githubusercontent.com/6872571/174438168-bd58277c-b4fc-4d23-a34a-19fb56fa3265.png)

**Comment by dougburks** [comment, marked as answer]
I'm not able to duplicate this using Google Chrome or chromium.

From https://docs.securityonion.net/en/2.3/soc.html:
> We recommend chromium or chromium-based browsers such as Google Chrome. Other browsers may work, but fully updated chromium-based browsers provide the best compatibility.
